lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -211,58 +213,94 @@ def test_resume_course_visibility(self):
 
 
 @attr(shard=1)
+@ddt.ddt
 class CourseInfoTitleTestCase(LoginEnrollmentTestCase, ModuleStoreTestCase):
     """
-    Tests of the CourseInfo page title.
+    Tests of the CourseInfo page title site configuration options.
     """
-
     def setUp(self):
         super(CourseInfoTitleTestCase, self).setUp()
-        self.course = CourseFactory.create()
-        self.page = ItemFactory.create(
-            category="course_info", parent_location=self.course.location,
-            data="OOGIE BLOOGIE", display_name="updates"
+        self.course = CourseFactory.create(
+            org="HogwartZ",
+            number="Potions_3",
+            display_organization="HogwartsX",
+            display_coursenumber="Potions101",
+            display_name="Introduction to Potions"
         )
 
-    def test_info_title(self):
+    @ddt.data(
+        # Default site configuration shows course number, org, and display name as subtitle.
+        (dict(),
+         "Welcome to HogwartsX's Potions101!", "Introduction to Potions"),
```

Comment:
#NIT add a space after this line to make it consistent with rest of the DDT data set